docs/esa/jwst/jwst.rst

```diff
@@ -225,69 +224,67 @@ To query the data products associated with a certain Observation ID
 .. code-block:: python
 
   >>> from astroquery.esa.jwst import Jwst
-  >>> product_list=Jwst.get_product_list(observation_id='jw00777011001_02104_00001_nrcblong')
+  >>> product_list=Jwst.get_product_list(observation_id='jw01063107001_02101_00013_nrca3')
```

Comment:
It would be better if the code examples followed [PEP 8](https://peps.python.org/pep-0008/). In particular [`=` in variable assignment should be surrounded with spaces](https://peps.python.org/pep-0008/#other-recommendations). I'm highlighting it only here, but there are more instances of this in the file.
```suggestion
  >>> product_list = Jwst.get_product_list(observation_id='jw01063107001_02101_00013_nrca3')
```